test/e2e/specs/specs-playwright/wp-start__importer.ts

```diff
@@ -14,23 +14,129 @@ describe( DataHelper.createSuiteTitle( 'Site Import' ), function () {
 		startImportFlow = new StartImportFlow( page );
 	} );
 
-	it( 'Log in', async function () {
+	// Login in default page.
+	it( 'Log in', async () => {
 		const loginPage = new LoginPage( page );
 		await loginPage.login( { account: 'defaultUser' } );
 	} );
 
-	describe( 'Follow the WordPress flow', function () {
-		it( 'Navigate to Capture page', async function () {
-			await page.goto(
-				DataHelper.getCalypsoURL( '/start/importer', { siteSlug: 'e2eflowtesting4.wordpress.com' } )
-			);
+	/**
+	 * Navigate to initial setup page
+	 *
+	 * @param siteSlug The site slug URL.
+	 */
+	const navigateToSetup = ( siteSlug = 'e2eflowtesting4.wordpress.com' ) => {
+		it( `Navigate to Setup page as ${ siteSlug }`, async () => {
+			await startImportFlow.startSetup( siteSlug );
 			await startImportFlow.validateURLCapturePage();
 		} );
+	};
+
+	// A normal and valid import flow.
+	describe( 'Follow the import flow', () => {
+		navigateToSetup();
 
 		it( 'Start a WordPress import', async () => {
 			await startImportFlow.enterURL( 'make.wordpress.org' );
 			await startImportFlow.validateScanningPage();
 			await startImportFlow.validateImportPage();
+			await startImportFlow.clickButton( 'Import your content' );
+
+			// When the new flows will be created this check will need to be replaced
+			await startImportFlow.validateMigrationPage( 'https://make.wordpress.org/' );
+		} );
+	} );
+
+	// An import flow that redirect to the "start building" page.
+	describe.each( [
+		// wordpress.com is already a WPCOM site and ever will be.
+		{ url: 'wordpress.com', reason: 'Your site is already on WordPress.com' },
+		// example.com will never be a WordPress site.
+		{ url: 'example.com', reason: "Your existing content can't be imported" },
+	] )( "Follow the WordPress can't be imported flow", ( { url, reason } ) => {
+		navigateToSetup();
+
+		it( `Start an invalid WordPress import (${ reason })`, async () => {
+			await startImportFlow.enterURL( url );
+			await startImportFlow.validateScanningPage();
+			await startImportFlow.validateBuildingPage( reason );
+			await startImportFlow.startBuilding();
+			await startImportFlow.validateDesignPage();
+		} );
```

Comment:
Typically, it would be preferable to break up some of these nested `describe` blocks into their own spec files to take advantage of parallelization (given that subsequent steps don't rely on previous ones). 

With that said though, it looks like these tests run very quickly (according to TC) and so this is not a pressing concern at all - more of a FYI.